build-logic/src/main/java/qsl/internal/dependency/QslLibraryDependency.java

```diff
@@ -0,0 +1,67 @@
+package qsl.internal.dependency;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.gradle.api.Named;
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.Dependency;
+import org.gradle.api.plugins.JavaPlugin;
+import org.jetbrains.annotations.NotNull;
+
+public class QslLibraryDependency  implements Named {
```

Comment:
Is github being wierd or is there a double space after the class name?

Reply:
there is.